backend/lambda-functions/deleteSessions/main.go

```diff
@@ -16,6 +18,11 @@ func main() {
 		return
 	}
 
+	H.SetProjectID("1jdkoe52")
```

Comment:
@Vadman97 just added these lines here to start logging, anything else we need to do?

Reply:
looks good! could we put this in the other lambdas too? you could also put this in `NewHandlers()` but then the `Stop()` won't be easy to add, so maybe just putting it in all the lambdas' `main` fns makes sense.